### PR TITLE
Add code generation test for aarch64

### PR DESCRIPTION
**Pull request by hadusam**
Related #335 
Only added code generation tests.
After this PR, I will add binary execution tests on raspberry-pi(aarch64).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by iizukak** [comment]
@primenumber This PR looks what you want.

File: dlk/tests/test_code_generation.py [hunk 1]
```diff
@@ -331,18 +344,22 @@ def codegen_cpu(self,
...
+            cmake_defs += [cmake_use_aarch64, cmake_use_neon]
+        elif cpu_name == 'arm':
             cmake_defs += [cmake_use_arm, cmake_use_neon]
-        if cpu_name == 'arm_fpga':
+        elif cpu_name == 'arm_fpga':
             cmake_defs += [cmake_use_arm, cmake_use_neon, cmake_use_fpga]
-        if use_avx == True:
-            cmake_defs += [cmake_use_avx]
+        elif cpu_name == 'x86_64':
+            if use_avx == True:
```
Comment (tsawada) [inline dlk/tests/test_code_generation.py]:
```suggestion
            if use_avx:
```

File: dlk/tests/test_code_generation.py [hunk 2]
```diff
@@ -366,9 +383,11 @@ def codegen_cpu(self,
         if not use_run_test_script:
             if cpu_name == 'x86_64':
                 percent_failed = self.run_library(generated_lib, input_path, expected_output_path)
-            else:
+            elif cpu_name == 'arm' or cpu_name == 'arm_fpga':
                 percent_failed = \
                     self.run_library_on_remote(FPGA_HOST, output_path, generated_lib, input_path, expected_output_path)
+            else:
+                return
```
Comment (tsawada) [inline dlk/tests/test_code_generation.py]: what do we expect here?
maybe we should do:
```
elif cpu_name == 'aarch64':
    # comment explaining why
    return
eles:
    self.fail("Unexpected cpu_name: %s" % cpu_name)
```

Comment (hadusam) [reply dlk/tests/test_code_generation.py]: Thanks, I updated! 👍

**Comment by iizukak** [comment]
Will merge after test passed.

**Pull request merged**